libraries/config/include/config/state_api_config.hpp

```diff
@@ -52,8 +52,8 @@ struct DPOSConfig {
   uint16_t max_block_author_reward = 0;
   uint16_t commission_change_delta = 0;
   uint32_t commission_change_frequency = 0;  // number of blocks
-  uint32_t delegation_delay = 0;             // number of blocks
-  uint32_t delegation_locking_period = 0;    // number of blocks
+  uint32_t delegation_delay = 5;             // number of blocks
+  uint32_t delegation_locking_period = 5;    // number of blocks
```

Comment:
Not sure why we was setting this to zero by default, but I think it should be more correct to set it to 5 here. At least for some reason configs for py_tests is generated from this default values, but not from some json config 
@JakubFornadel what do you think about this?

Reply:
no idea